CAFAna/Core/SignalHandlers.cxx

```diff
@@ -0,0 +1,125 @@
+// This file sets things up (when libCAFAnaCore is loaded) so that we invoke
```

Comment:
I think this comment should now reference libCAFAnaCoreExt?

Reply:
ah, yes.
1dd934e97a14d815edd17e8184f06534a40fc8f4